map/place_page_info.hpp

```diff
@@ -32,6 +32,13 @@ enum class SponsoredType
   Geochat
 };
 
+enum class LocalAdsStatus
+{
+  Unavailable,
```

Comment:
https://english.stackexchange.com/questions/50276/the-service-is-temporarily-unavailable-vs-not-available